test/cli/testdata/cnquery_scan.ct

```diff
@@ -19,21 +19,21 @@ Available Commands:
   aws              Scan an AWS account
   azure            Scan an Azure subscription
   container        Scan a running container or container image
-  docker           Scan a running Docker container or Docker image
+  docker           Scan a running Docker container or image
```

Comment:
the test fails right now, since we do not compile all providers locally yet. therefore we need to split out the cli testing into a separate PR for now. Once the providers are released, the updated tests are required.